CMakeLists.txt

```
@@ -2566,11 +2566,20 @@ install(FILES
         COMPONENT netdata
         DESTINATION usr/lib/netdata/system/systemd)
 
+install(FILES
+        system/systemd/journald@netdata.conf
+        COMPONENT netdata
+        DESTINATION usr/lib/netdata/system/systemd)
+
 if(BUILD_FOR_PACKAGING)
         install(FILES
                 ${CMAKE_BINARY_DIR}/system/systemd/netdata.service
                 COMPONENT netdata
                 DESTINATION lib/systemd/system)
+        install(FILES
+                system/systemd/journald@netdata.conf
+                COMPONENT netdata
+                DESTINATION lib/systemd/system)
```

```suggestion
        install(DIRECTORY
                COMPONENT netdata
                DESTINATION lib/systemd/journald@netdata.conf.d
        install(FILES
                system/systemd/journald@netdata.conf
                COMPONENT netdata
                DESTINATION lib/systemd/journald@netdata.conf.d/netdata.conf)
```

This should be all that’s required to make the native packages work correctly.